README.md

```diff
@@ -13,6 +13,30 @@ The following table details orchestration platforms suitable for deployment of t
 |Red Hat® OpenShift®|4.7|x86|
 |Red Hat OpenShift|4.6|x86|
 
+# Prerequisites
+
+To use the Block Storage for VPC driver, complete the following tasks:
```

Comment:
Please mention that we need to pull/push image from public registry  i.e icr.io/ibm. As this this the path we are using in controller and node-server yamls.

Reply:
Thanks @sameshai for the comment. This is already mentioned in the README
https://github.com/IBM/ibm-vpc-block-csi-driver

current PR is an enhancement on top of the existing one.